meta-zarhus-distro/recipes-zarhus/packagegroups/packagegroup-zarhus.bb

```diff
@@ -23,4 +23,6 @@ RDEPENDS:${PN}-dbg = " \
     libgpiod \
     libgpiod-tools \
     devmem2 \
+    libdrm \
```

Comment:
Should it be in `-dbg` package? This is described as `Userspace interface to the kernel DRM services`, so you can add it to `-graphics` package for example.

Reply:
I wonder if we need explicitly install it here in the first place.

Reply:
In manifest there is `libdrm2` package.
I wanted libdrm mostly for `modeset` and other testing tools. Manifest with this change contains:

```
libdrm-amdgpu1 cortexa55 2.4.120
libdrm-etnaviv1 cortexa55 2.4.120
libdrm-tests cortexa55 2.4.120
libdrm2 cortexa55 2.4.120
```

Reply:
Removed `libdrm`, works with only `libdrm-tests`